### PR TITLE
Digital Silver Token & Digital Diamond Token Reg Submission

### PR DESCRIPTION
**Pull request by DigitalGold2022**
# Pull Request Template

## Description

Registration request for the Digital Silver Token

## Type of change

- [x] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

**Review by salahdaghestani — APPROVED** [review]

**Comment by DigitalGold2022** [comment]
I am sorry I messed up the pull request.  i am new to github my apologies.  I was trying to send 2 different pull requests.  One for Digital Silver token and one for Digital Diamond token.  Did I mess up the process

File: 59515c449f1ec1d21a4b2d93013337d13b29d74c3a665a11d5b7fc6644534c5652.json
Comment (cleanerm5) [inline 59515c449f1ec1d21a4b2d93013337d13b29d74c3a665a11d5b7fc6644534c5652.json:1]: this file must be removed from the PR

**Review by DigitalGold2022 — COMMENTED** [review]
I have delete the file.  I do apologize for all the confusion.  We are new to Github and are trying two submit 2 sets of metadata.  One for the Digital Silver token and one for the Digital Diamond token.  Any help with this is great.  TY

File: 59515c449f1ec1d21a4b2d93013337d13b29d74c3a665a11d5b7fc6644534c5652.json
Comment (DigitalGold2022) [reply 59515c449f1ec1d21a4b2d93013337d13b29d74c3a665a11d5b7fc6644534c5652.json:1]: I think I deleted the correct one.  I hope

Comment (DigitalGold2022) [reply 59515c449f1ec1d21a4b2d93013337d13b29d74c3a665a11d5b7fc6644534c5652.json:1]: Thank for the help!  Im a little bit lost

**Comment by DigitalGold2022** [comment]
anyone know the status

**Comment by DigitalGold2022** [comment]
upload 595 request to pr

**Review by DigitalGold2022 — COMMENTED** [review]
uploaded file

**Review by cleanerm5 — APPROVED** [review]

**Pull request merged**